### PR TITLE
Adding messaging regarding not supporting React Router Loaders & Actions

### PR DESCRIPTION
**Pull request by JacobMGEvans**

### CODE REVIEW

**Review by nexus-codingg — APPROVED** [review]
this LGTM 🫡

File: docs/references/react/add-react-router.mdx [hunk 1]
```diff
@@ -33,6 +33,10 @@ description: Learn how to add React Router to your React application using their
 
 # Add React Router to your Clerk-powered React application
 
+<Callout type="warning">
+  Loaders and Actions are currently not supported by Clerk at the moment with React Router.
```
Comment (jescalan) [inline docs/references/react/add-react-router.mdx:37]:
```suggestion
  [Loaders](https://reactrouter.com/en/main/route/loader) and [Actions](https://reactrouter.com/en/main/route/action) are currently not supported by Clerk at the moment with React Router.
```

**Review by jescalan — APPROVED** [review]

File: docs/references/react/add-react-router.mdx [hunk 2]
```diff
@@ -33,6 +33,10 @@ description: Learn how to add React Router to your React application using their
 
 # Add React Router to your Clerk-powered React application
 
+<Callout type="warning">
+  [Loaders](https://reactrouter.com/en/main/route/loader) and [Actions](https://reactrouter.com/en/main/route/action) are currently not supported by Clerk at the moment with React Router.
+</Callout>
+
```
Comment (royanger) [inline docs/references/react/add-react-router.mdx:39]: The callout content is good. However please update this with the callout markdown from the CONTRIBUTING guide.

File: docs/references/react/add-react-router.mdx [hunk 3]
```diff
@@ -33,6 +33,10 @@ description: Learn how to add React Router to your React application using their
 
 # Add React Router to your Clerk-powered React application
 
+<Callout type="warning">
+  [Loaders](https://reactrouter.com/en/main/route/loader) and [Actions](https://reactrouter.com/en/main/route/action) are currently not supported by Clerk at the moment with React Router.
```
Comment (royanger) [inline docs/references/react/add-react-router.mdx:37]:
```suggestion
  [Loaders](https://reactrouter.com/en/main/route/loader) and [Actions](https://reactrouter.com/en/main/route/action) are currently not supported by Clerk with React Router.
```
I would trim this. I had a chat with Bryce last night about this and we have no plans to support these at the moment.

**Review by royanger — CHANGES_REQUESTED** [review]
Just a couple of minor changes needed here.

**Review by royanger — APPROVED** [review]
🚀

**Pull request merged**